pytket/qir/conversion/qirgenerator.py

```diff
@@ -113,6 +119,60 @@
     BitZero: 0,
 }
 
+_TK_CLEXPR_OP_TO_PYQIR: dict = {
+    ClOp.BitAnd: lambda b: b.and_,
+    ClOp.BitOr: lambda b: b.or_,
+    ClOp.BitXor: lambda b: b.xor,
+    ClOp.BitEq: lambda b: partial(b.icmp, IntPredicate.EQ),
+    ClOp.BitNeq: lambda b: partial(b.icmp, IntPredicate.NE),
+    # ClOp.BitNot
```

Comment:
Are all of this ops implemented later on differently? Or are some of them not supported?

Reply:
Some are implemented differently; a couple of them are unsupported (these are `RegDiv` and `RegPow` which are also unsupported for `logic_exp` conversion; there is an issue for this).